inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/es/ElasticsearchDataNodeDTO.java

```diff
@@ -85,8 +85,8 @@ public static ElasticsearchDataNodeDTO getFromJson(@NotNull String extParams) {
         try {
             return JsonUtils.parseObject(extParams, ElasticsearchDataNodeDTO.class);
         } catch (Exception e) {
-            LOGGER.error("Failed to extract additional parameters for Elasticsearch data node: ", e);
```

Comment:
Does the `e.getMessage()` show the detail of the parse error? Such as which field was parsed failed.

If not, then I think this error log is necessary.

Reply:
API calls will be more frequent. If every request is logged, the manager node will explode due to too much log output
Then the exception of this block is the json encoding and decoding operation of the extParams content. If it fails, the caller will provide the specific content.

Reply:
I did not understand that "If it fails, the caller will provide the specific content."

This extParams parameter is saved in the DB. In case of wrong saving in the DB, or the target conversion class changes, which leads to parsing failure, can you get the specific information of extParams from "e.getMessage"?

If you can't get it, you need to check the specific information in the DB according to the previous series of logs. This is undoubtedly more difficult.

Reply:
The modification of this place is to allow the caller to analyze the problem, rather than the system administrator to go to the log to analyze the problem

Specifically for this error, the error is the parameter decoding error of extParams, mainly two types, one is that the data passed by the caller is not in json format, and the other is the content of the class that does not match the class specified by the parser.

As the caller, it is clear what content is passed. Based on this basis, the error code and error information provided by the assistant are very clear about which field has an error. Through the specified parsing error information and the original data of the call, the call The operator can analyze the problem by himself, without the need for the system maintainer to view the log, that is, this log output is unnecessary.

Reply:
I think our discussion may be a little inconsistent.
**It is correct to throw an exception, but my question is, can "e.getMessage" throw the details of the error to the caller?**

Reply:
You can simulate and throw an exception directly at the above modification point, to see if the error message of the exception can be obtained in the response